recipes/freetype/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+            --with-zlib=$PREFIX \
+            --with-png=yes \
+            --with-bzip2=no \
+            --without-harfbuzz
```

Comment:
You could consider about changing this option, now that we packages for harfbuzz.